### PR TITLE
fit:removed % Done bar

### LINKED ISSUE #2046: Remove % done bar from UI and code
**Is your feature request related to a problem? Please describe.**
It's very annoying when the screen is cluttered. The % done was a nice experiment, but it's better to visually implement this on the existing day bar itself.

**Describe the solution you'd like**
This issue is just for removing the %done bar from the UI and code.  

**Describe alternatives you've considered**
We're keeping it - but will find a way to superpose this information on the existing day bar.

**Additional context**
You can see the grey %done bar here:  
image

### PR DESCRIPTION
**Pull request by codervinod123**
Resolves #2046

### CODE REVIEW

**Comment by tijlleenders** [comment]
@codervinod123 Is this ready for review?  
The grey bar can also be removed, not only the %done text.

**Comment by codervinod123** [comment]
> @codervinod123 Is this ready for review? The grey bar can also be removed, not only the %done text.
@tijlleenders let me remove

**Comment by codervinod123** [comment]
@tijlleenders please review

**Review by tijlleenders — APPROVED** [review]
@codervinod123 From UI side this looks great.  

Has all dead code been removed too? So like a CSS classes/variables/components/etc that were only used by the 'Done' component, and are now 'dead code'?  

There's a lot of code formatting changes too. Maybe @vinaybadgujar102 can advise what the standard formatter is for our project and how to implement this. Otherwise it makes it hard to review the code, as formatting changes are mixed in with 'real' changes to the code...

**Comment by codervinod123** [comment]
@tijlleenders 

All dead code been removed.
There's a lot of code formatting changes just because of linting and prettier rule for this code base

File: src/components/MyTimeComponents/ColorBands.tsx [hunk 1]
```diff
@@ -14,20 +14,16 @@ interface IColorBands {
 }
 
 const ColorBands: React.FC<IColorBands> = ({ list, tasksStatus, day, active }) => {
```
Comment (vinaybadgujar102) [inline src/components/MyTimeComponents/ColorBands.tsx:16]: remove unnecessary props

**Review by vinaybadgujar102 — CHANGES_REQUESTED** [review]

File: src/components/MyTimeComponents/ColorBands.tsx [hunk 2]
```diff
@@ -41,31 +37,11 @@ const ColorBands: React.FC<IColorBands> = ({ list, tasksStatus, day, active }) =
     }
   });
```
[the remark this answers is not in the document]
Comment (codervinod123) [reply src/components/MyTimeComponents/ColorBands.tsx:38]: got it , Thanks

[the remark this answers is not in the document]
Comment (vinaybadgujar102) [reply src/components/MyTimeComponents/ColorBands.tsx:37]: hey @codervinod123. I noticed that correct lines are not highlighted by me in this comment. Sorry for the inconvenience. 
You can remove the code from line 17 to 37 which includes all business logic. Then remove the unused props.

**Comment by codervinod123** [comment]
@vinay can you review  https://github.com/tijlleenders/ZinZen/pull/2047/commits/f5a8f5cc839bd17aecbfc711b323f519d9e575fd

**Review by vinaybadgujar102 — APPROVED** [review]

**Comment by vinaybadgujar102** [comment]
@codervinod123 Thanks for the contribution!

**Pull request merged**